api/src/apps/routes.rs

```diff
@@ -766,6 +836,48 @@ mod tests {
             let response = get.dispatch().await;
             assert_eq!(response.status(), Status::BadRequest);
         }
+
+        #[tokio::test]
+        async fn render_log_weblink_with_no_limit() -> Result<(), crate::apps::AppsServiceError> {
+            let (host_meta_cache, mut _host_meta_crawler) = crate::host_meta_crawling();
+
+            let client =
+                set_up_rocket_with_dummy_infrastructure_and_a_running_app(host_meta_cache).await?;
+
+            let response = client
+                .get("/api/apps/master/logs/service-a")
+                .header(ContentType::Text)
```

Comment:
Wrong header here in the context of a client. [A client sends the accept header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept):

> The Accept request HTTP header indicates which content types, expressed as [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types), the client is able to understand. The server uses [content negotiation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation) to select one of the proposals and informs the client of the choice with the [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) response header. 

When you change it to the Accept header it should fail because you also modified the content-type in the production code.